scripts/install/configuration/replace_ssh_keys.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+usage()
+{
+   # Display Help
+   echo "Replaces the ssh keys in Kubernetes"
+   echo "NOTE: This does not update deployed keys"
+   echo
+   echo "Usage: replace_ssh_keys.sh [ --public-key-file file ]"
+   echo "                            [ --private-key-file file ]"
```

Comment:
Maybe delete one of the spaces so that the arguments line up vertically.

Reply:
Fixed